docs/docusaurus.config.js

```diff
@@ -44,46 +65,23 @@ const config = {
 	plugins: [
 		'@docusaurus/theme-live-codeblock',
 		[
-			'docusaurus-plugin-typedoc',
-
-			// Plugin / TypeDoc options
+			'docusaurus-plugin-typedoc-api',
 			{
-				entryPoints: [
-					'../packages/web3',
-					'../packages/web3-common',
-					'../packages/web3-core',
-					'../packages/web3-errors',
-					'../packages/web3-eth',
-					'../packages/web3-eth-abi',
-					'../packages/web3-eth-accounts',
-					'../packages/web3-eth-contract',
-					'../packages/web3-eth-ens',
-					'../packages/web3-eth-iban',
-					'../packages/web3-eth-personal',
-					'../packages/web3-net',
-					'../packages/web3-providers-http',
-					'../packages/web3-providers-ws',
-					'../packages/web3-providers-ipc',
-					'../packages/web3-utils',
-					'../packages/web3-validator',
-				],
-				entryPointStrategy: 'packages',
-				cleanOutputDir: true,
-				sidebar: {
-					fullNames: true,
+				projectRoot: join(__dirname, '..'),
+				// Monorepo
+				packages: packages.map(p => `packages/${p}`),
+				minimal: false,
```

Comment:
Its by default false for rendering minimal layout

Reply:
Yes, but I want to keep it mention explicitly for readability and understanding.